apps/shell/tash_command.c

```diff
@@ -269,6 +269,68 @@ int tash_execute_cmd(char **args, int argc)
 	return 0;					/* Need to pass the appropriate error later */
 }
 
+/** @name tash_tab_cmd
+ * @brief API to TASH tab commands
+ * @ingroup tash
+ * @param[in] cmd - command's first string
+ * @param[in] pos - length of command's first string
+ * @param[in] pressed_tab: true - tab is pressed more than once
+						   false - tab is pressed once
+ * @retval true Success
+ * @retval false Failure
+ *
+ * Example Usage:
+ * @code
+ *
+ * @endcode
+ */
+bool tash_tab_cmd(char *cmd, int *pos, bool pressed_tab)
```

Comment:
I think this function can be static. If then, we don't need to declare prototype in tash_internal.h